src/main/java/fr/xephi/authme/command/executable/authme/MessagesCommand.java

```diff
@@ -22,15 +16,6 @@
  */
 public class MessagesCommand implements ExecutableCommand {
```

Comment:
I also need to update this command so that it doesn't take "help" as second argument anymore.